### DISCUSSION TITLE
Cannot create mail server

### DISCUSSION THREAD

**Discussion by SPEED0U** [postalserver/postal#2520] [Help with using Postal]
Hello,

I'm having an error 500 (in console) when attempting to create a mail server.
I already tested mysql connection, it is fine. 

And I did not find where are the logs, docker logs are not helpful.
Checked for similar posts, didn't help me.

Thanks you.

**Comment by SPEED0U** [comment]
I could find the error, it tells me it cannot connect with the sql user and password I've set, the thing is with those credentials I can connect thru PhpMyAdmin.
Is it because my mysql instance is not under docker ? As I already got one running I prefered using this one rather than having an instance under docker.

**Comment by willpower232** [comment]
Can you share a screenshot of the error message you are seeing?
